app/components/ui/PopUpMessage.scss

```diff
@@ -0,0 +1,41 @@
+.pop-up-message {
+  display: table;
```

Comment:
Do you know if this is necessary? I just never use `display: table`, so I'm curious what it is doing in this context.

Reply:
Haha, yes. Would love your help on how to do this properly. I was trying to center text vertically in the pop-up message, and after a ton of experimenting and failing, I found a StackOverflow comment about using `display: table` to perform vertical centering.

I'll bet there is something much better, like `display: flex` or similar.

Would you recommend `display: flex`? Or is there an even better approach?

In the meantime, I'll try to find a better alternative and push another attempt

Reply:
Ahh gotcha! Ya, display: flex with align-items: center should do the trick! (It might be justify-content, I always mix these up...)